README.md

```diff
@@ -214,30 +380,11 @@ There are two use cases for deep linking that can be relevant here:
 
```

Comment:
I would remove lines 376 to 379 above. I don't think those are necessary if we add the description above.

Reply:
Done.